### PR TITLE
Selectively decide access tree depth + better access level dropdown

### PR DESCRIPTION
**Pull request by kitallis**
**Story cards**

* https://app.clubhouse.io/simpledotorg/story/850/as-a-manager-or-power-user-i-can-view-all-the-other-admins-in-my-organization-and-edit-them

* https://app.clubhouse.io/simpledotorg/story/850/as-a-manager-or-power-user-i-can-view-all-the-other-admins-in-my-organization-and-edit-them

* https://app.clubhouse.io/simpledotorg/story/1107/the-access-tree-for-admins-should-only-show-the-tree-depths-as-much-as-necessary

## Because

This work was originally split up into 3 PRs, it was too difficult to merge them and a huge waste of time so I've combined them all in here.

## This addresses

This does the following:

* Adds a way to list all admins in your own organization
* Show the accesses of a particular admin
* Change the depth of the tree based on the scope of your access

### CODE REVIEW

File: app/assets/stylesheets/partials/_invite_admin.scss [hunk 1]
```diff
@@ -6,6 +6,68 @@
   transition: $duration;
```
Comment (kitallis) [inline app/assets/stylesheets/partials/_invite_admin.scss:6]: Rename this file to `admin_form.scss`.

File: app/controllers/admins_controller.rb [hunk 2]
```diff
@@ -99,7 +113,9 @@ def set_admin
   end
 
   def 🆕set_admin
-    @admin = authorize1 { current_admin.accessible_admins(:manage).find(params[:id]) }
+    if Flipper.enabled?(:new_permissions_system_aug_2020, current_admin)
```
Comment (kitallis) [inline app/controllers/admins_controller.rb:116]: Remove this check, it's redundant.

File: app/controllers/email_authentications/invitations_controller.rb [hunk 3]
```diff
@@ -102,7 +102,7 @@ def user_params
   end
 
   def selected_facilities
-    params[:facilities]
+    params[:facilities].flatten
```
Comment (kitallis) [inline app/controllers/email_authentications/invitations_controller.rb:105]: Try to get rid of this `flatten`.

File: app/presenters/admin_access_presenter.rb [hunk 4]
```diff
@@ -8,13 +10,39 @@ def initialize(admin)
     super
   end
 
+  def display_access_level
+    OpenStruct.new(UserAccess::LEVELS.fetch(admin.access_level.to_sym))
+  end
+
   def permitted_access_levels_info
```
Comment (kitallis) [inline app/presenters/admin_access_presenter.rb:17]: Just call it `permitted_access_levels`.

File: app/views/email_authentications/invitations/_access_tree.html.erb [hunk 5]
```diff
@@ -1,72 +1,28 @@
-<div class="card access-tree">
```
Comment (kitallis) [inline app/views/email_authentications/invitations/_access_tree.html.erb:1]: Rename this to `_organization_access_tree`.

**Review by vkrmis — APPROVED** [review]

**Pull request merged**